docs-site/content/guide/laravel-full-text-search.md

```diff
@@ -0,0 +1,567 @@
+            TYPESENSE_ENABLE_CORS: '${TYPESENSE_ENABLE_CORS:-true}'
+        volumes:
+            - 'sail-typesense:/typesense-data'
+        networks:
+            - sail
+        healthcheck:
+            test:
+                - CMD
+                - wget
+                - '--no-verbose'
+                - '--spider'
+                - 'http://localhost:8108/health'
+            retries: 5
+            timeout: 7s
+```
+</template>
+
+</Tabs>
+
+In order for the changes to take effect, let's rebuild the Docker containers:
+
+<Tabs :tabs="['Shell']">
+
+<template v-slot:Shell>
+
+```shell
+./vendor/bin/sail down
+
+./vendor/bin/sail up -d
+```
+
+</template>
+
+</Tabs>
+
+## Step 3: Install Laravel Scout
+
+As per [Laravel Sail's documentation](https://laravel.com/docs/11.x/sail#executing-composer-commands), to install Laravel Scout via Composer, let's run the following command:
+
+<Tabs :tabs="['Shell']">
+
+<template v-slot:Shell>
+
+```shell
+./vendor/bin/sail composer require laravel/scout
+```
+
+</template>
+
+</Tabs>
+ 
+You'll also need to install the [Official PHP client for the Typesense API](https://github.com/typesense/typesense-php) by running the following command:
+
+<Tabs :tabs="['Shell']">
+
+<template v-slot:Shell>
+
+```shell
+./vendor/bin/sail composer require php-http/curl-client typesense/typesense-php
+```
+
+</template>
+
+</Tabs>
+
+
+Next, let's publish the Laravel Scout configuration file:
+
+<Tabs :tabs="['Shell']">
+
+<template v-slot:Shell>
+
+```shell
+./vendor/bin/sail artisan vendor:publish --provider="Laravel\Scout\ScoutServiceProvider"
+```
+
+</template>
+
+</Tabs>
+
+And configure Laravel Scout to use the Typesense driver by modifying the `config/scout.php` file:
+
+<Tabs :tabs="['PHP']">
+
+<template v-slot:PHP>
+
+```php
+...
+return [
+    ...
+    'driver' => env('SCOUT_DRIVER', 'typesense'),
+    ...
+];
+```
+
+</template>
+
+</Tabs>
+
+## Step 4: Create a Model and Migrate Data
+
+Next, let's add some data to the PostgreSQL database. You can use any dataset you want, but this guide will use Terenci Claramunt's ([@terencicp](https://github.com/terencicp)) public dataset of Steam Games released from 2013 to 2023, which you can find [here](https://github.com/typesense/showcase-laravel-steam-games-search/blob/master/data/games.csv), and save it in the `data` folder.
+To use it, you'll have to create an Eloquent model for the Steam Games, and create a migration file, along with a model file and a controller. To do so, let's use the following command:
+
+<Tabs :tabs="['Shell']">
+
+<template v-slot:Shell>
+
+```shell
+
+./vendor/bin/sail artisan make:model Game -mrc
+
+```
+
+</template>
+
+</Tabs>
+
+This command will create a new Eloquent model named `Game`, along with a migration file and a resource controller. Let's now modify the model file `app/Models/Game.php` to include the necessary columns for the Steam Games dataset:
+
+<Tabs :tabs="['PHP']">
+
+<template v-slot:PHP>
+
+```php{18,25-36,43-53}
```

Comment:
These highlights seem to be off by a few lines. Could you run the site locally and verify?

Reply:
This is how it looks on my end:
![image](https://github.com/typesense/typesense-website/assets/93339170/ced4d485-0f7c-4b06-8226-d0be2e3325ee) 

Should I shift it up or down?

Reply:
Hmm, it looks fine here. It extended a couple of lines further down with each block...